src/Microsoft.AspNetCore.StaticFiles/StaticFileContext.cs

```diff
@@ -231,60 +230,8 @@ private void ComputeRange()
                 return;
             }
 
-            var rawRangeHeader = _request.Headers[HeaderNames.Range];
-            if (StringValues.IsNullOrEmpty(rawRangeHeader))
-            {
-                return;
-            }
-
-            // Perf: Check for a single entry before parsing it
-            if (rawRangeHeader.Count > 1 || rawRangeHeader[0].IndexOf(',') >= 0)
-            {
-                // The spec allows for multiple ranges but we choose not to support them because the client may request
-                // very strange ranges (e.g. each byte separately, overlapping ranges, etc.) that could negatively
-                // impact the server. Ignore the header and serve the response normally.
-                _logger.LogMultipleFileRanges(rawRangeHeader.ToString());
```

Comment:
No, because then you'll have to add LogMultipleFileRanges (which you should now remove).